pom.xml

```diff
@@ -88,11 +88,13 @@
         <artifactId>nukleus-maven-plugin</artifactId>
         <version>${nukleus.plugin.version}</version>
         <configuration>
-          <scopeNames>http</scopeNames>
+          <scopeNames>core http</scopeNames>
+          <packageName>org.reaktivity.nukleus.http.internal.types</packageName>
```

Comment:
`packageName` should generate into a package you own, such as `org.reaktivity.specification.http.internal.types`.